src/components/DarkModeToggle.tsx

```diff
@@ -0,0 +1,52 @@
+import "./ToolIcon.scss";
+
+import React from "react";
+import { t } from "../i18n";
+
+export type Appearence = "light" | "dark";
+
+// We chose to use only explicit toggle and not a third option for system value,
+// but this could be added in the future.
+export const DarkModeToggle = (props: {
+  value: Appearence;
+  onChange: (value: Appearence) => void;
+}) => {
+  return (
+    <label
+      className={`ToolIcon ToolIcon__lock ToolIcon_type_floating ToolIcon_size_M`}
+      title={t("buttons.toggleDarkMode")}
+    >
+      <input
+        className="ToolIcon_type_checkbox ToolIcon_toggle_opaque"
+        type="checkbox"
+        onChange={(event) =>
+          props.onChange(event.target.checked ? "dark" : "light")
+        }
+        checked={props.value === "dark"}
+        aria-label={t("buttons.toggleDarkMode")}
+      />
+      <div className="ToolIcon__icon">
+        {props.value === "light" ? ICONS.MOON : ICONS.SUN}
+      </div>
+    </label>
+  );
+};
+
+const ICONS = {
+  SUN: (
```

Comment:
Suggestion: The current sun icon looks somewhat closer to settings icon so not sure if it will be intutive for user so I tried with below 
<img width="936" alt="Screenshot 2020-08-12 at 7 29 04 PM" src="https://user-images.githubusercontent.com/11256141/90024639-b704a800-dcd2-11ea-984a-18180c5cf4bc.png">
What do you think?

Reply:
Given people will see the moon first it's probably not a big concern. I explored this kind of icon but it looks more like "light" than "sun". But I'd be happy with either (so feel free to PR if you feel strongly).